config/routes.rb

```diff
@@ -65,7 +65,10 @@
   post 'admin/reset_2fa', to: 'admin#reset_2fa'
   post 'admin/email_all', to: 'admin#email_all'
 
-  resources :histories, only: [:create]
```

Comment:
Can we setup these routes the rails way like [this](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) instead?
```
resources :histories, only: [:create, :update, :destroy]
```

Reply:
@holmesie I spoke with @sgober about this and she mentioned that it wouldn't really make sense to call DELETE because we're not actually deleting the record, which is a fair point. I also think that it's pretty common in a lot of systems to also just mark records as deleted, so I was wondering what you think about this. Not a big deal at all, but just curious

Reply:
That makes sense - HTTP DELETE doesn't _necessarily_ mean an actual database `DELETE FROM `table_name` WHERE condition;`, but I understand if the controller looks that way it could get confusing. I'm honestly okay either way, as long as it's documented/commented clearly.

Reply:
thanks Adam, we can leave it the way it is then